util/src/main/java/org/datacommons/util/StatChecker.java

```diff
@@ -57,7 +57,7 @@
   // scaling factor, and unit of the stat var observations of the series summary.
   private final Map<String, SeriesSummary> seriesSummaryMap;
```

Comment:
Maybe its worth summarizing the concurrency handling of this class (spanning seriesSummaryMap, samplePlaces and svObsValues data structures)?  The stuff we discussed on chat...

Reply:
done